codes/1946_py3_sjskgks.py

```diff
@@ -0,0 +1,16 @@
+import sys
+input = sys.stdin.readline
+
+for case in range(int(input())):
+    numOfapp = int(input())
+    applicants = [list(map(int, input().split())) for _ in range(numOfapp)]
+    applicants.sort()
+    recruit = 1
+    limit = applicants[0][1]
+    for i in range(1, numOfapp):
+        if applicants[i][1]<limit:
+            limit = min(limit, applicants[i][1])

```

Comment:
for문을 다듬으면 더 좋을 것 같습니다.
첫번째 지원자를 보긴 하지만 부등호에 등호가 들어가 있지 않아 어차피 넘어가게 됩니다.
```suggestion
    for _, score in applicants:
        if score<limit:
            limit = min(limit, score)
```